multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -339,29 +337,26 @@ test.serial('lp deposits', async t => {
     proposal,
   });
 
-  await t.notThrowsAsync(() =>
-    retryUntilCondition(
-      () => fastLPQ(vstorageClient).metrics(),
-      ({ shareWorth }) =>
-        !isGTE(metricsPre.shareWorth.numerator, shareWorth.numerator),
-      'share worth numerator increases from deposit',
-      { log },
-    ),
+  await retryUntilCondition(
+    () => fastLPQ(vstorageClient).metrics(),
+    ({ shareWorth }) =>
+      !isGTE(metricsPre.shareWorth.numerator, shareWorth.numerator),
+    'share worth numerator increases from deposit',
+    { log },
   );
 
   const { useChain } = t.context;
   const queryClient = makeQueryClient(
     await useChain('agoric').getRestEndpoint(),
   );
 
-  await t.notThrowsAsync(() =>
-    retryUntilCondition(
-      () => queryClient.queryBalance(wallets['lp'], 'ufastlp'),
-      ({ balance }) => isGTE(toAmt(FastLP, balance), want.PoolShare),
-      'lp has pool shares',
-      { log },
-    ),
+  await retryUntilCondition(
+    () => queryClient.queryBalance(wallets['lp'], 'ufastlp'),
+    ({ balance }) => isGTE(toAmt(FastLP, balance), want.PoolShare),
+    'lp has pool shares',
+    { log },
   );
+  t.pass();
```

Comment:
How does a failure above propagate in the test results in the event of failure? "Unhandled error lp has pool shares condition not satisfied..."